packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -195,6 +195,32 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     }
   }
 
+  // Find next valid dropdown option
+  private _moveIndex(index: number, selectedIndex: number): number {
+    // Return selectedIndex if nothing has changed
```

Comment:
i'm confused. I reviewed this before. I suggested to rename this:

`_moveIndex(stepValue: number, ev: React.KeyboardEvent<HTMLElement>) { ... }`